src/pages/trade/ui/page.tsx

```diff
@@ -8,49 +8,145 @@ import { TradesTabs } from './trades-tabs';
 import { HistoryTabs } from './history-tabs';
 import { FormTabs } from './form-tabs';
 
-export const TradePage = () => {
+const sharedStyle = 'w-full border-t border-t-other-solidStroke overflow-x-hidden';
+
+// border-b border-b-other-solidStroke desktop:border-r desktop:border-r-other-solidStroke
```

Comment:
suggestion: can you fix the [desktop nav](https://github.com/penumbra-zone/dex-explorer/blob/main/src/widgets/header/ui/desktop-nav.tsx#L7) from shifting on page reloads?

https://github.com/user-attachments/assets/20cc6149-9f4f-484e-b1e8-1a1c844e96fb